runatlantis.io/docs/using-atlantis.md

```diff
@@ -76,7 +76,7 @@ atlantis apply
 atlantis apply -d .
 
 # Runs apply in the `project1` directory of the repo with workspace `default`
-atlantis apply -d project1
```

Comment:
this correct, it talks about the project Directory not the atlantis project

Reply:
Okay. This was a little confusing since `project1` could also be interpreted as the project name. I added an example of using `atlantis apply -p project1` to differentiate.

Reply:
the name of the project usually matches the name of the project in the atlantis.yaml.

Reply:
Okay. I see there is some consistency in using `project1` as the project directory https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html#project. I'll close this request.